server/chess.py

```diff
@@ -135,7 +135,13 @@ def validate_pawn_move(
                 return not self.get_piece(rank, file)
             elif absolute_file_delta == 1:
                 victim = self.get_piece(rank, file)
-                return victim and victim.side != pawn.side
+                en_passant_pawn = self.get_piece(pawn.rank, file)
+                en_passant_valid = (
+                    en_passant_pawn and en_passant_pawn.side != pawn.side
+                    and en_passant_pawn.first_move_last_turn
+                    and pawn.rank = (7 + pawn.side.forwards) / 2
```

Comment:
```suggestion
                    and pawn.rank == (7 - pawn.side.forwards) / 2
```
Missed an equals, also: forward for home is 1, `(7 + 1) / 2` is 4 but should be 3, `(7 - 1) / 2` is 3 but should be 4. Clever maths tricks are nice but only when they work, and it can be hard to tell at a glance if they are correct. Maybe just
```suggestion
                    and pawn.rank == (3 if pawn.side == models.Side.HOME else 4)
```
Would be better

Reply:
I say all this and then it's going to turn out you were right, lol

Reply:
Yeah I'm right 😂. The pawn that is taking needs to be on file 4 if home, and file 3 if away. I'm happy to make it more explicit though rather than the 7, if that will make it easier to read?

Reply:
Yeah you're right mb

Reply:
But yeah I think either make it more explicit or put a comment on the line before (`# ...` not `"""..."""` 😉)